src/theme/Navbar/Logo/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Logo from '@theme/Logo';
+import RenderOfficialContents from '@site/src/components/RenderOfficialContents';
+export default function NavbarLogo() {
```

Comment:
```suggestion
export const NavbarLogo = () => (
<Render...
```

Reply:
The components in the theme folder are third party components. Sometimes we need to swizzle (extract) them and only for some necessary adjustments. I think changing the export style might affect other node packages/components that import these swizzled components.